### ISSUE TITLE
Slow selection on 8.2Windows version 

### ISSUE THREAD

**Issue by Isaina** [ONLYOFFICE/DesktopEditors#1738]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Windows 10 / 8.1 / 7

### Version information

8.2

### Expected Behavior

Docx file , font is not installed on computer onlyoffice use replacement font and selecting text  need time and onlyoffice  freeze , when in task bar rise priority of onlyoffice executable to real time is much better and spell checker is active 

### Actual Behavior

Freeze onlyoffice program when selecting text

**Comment by Isaina** [comment]
Yes the font is not standard PT SansSent using the mail.com mail appOn 18.02.2025. at 08:56, Denis Myshaev wrote:
                
            
            
                
                    From: "Denis Myshaev" ***@***.***>Date: 18 February 2025To: "ONLYOFFICE/DesktopEditors" ***@***.***>Cc: "Mention" ***@***.***>,"Isaina" ***@***.***>Subject: Re: [ONLYOFFICE/DesktopEditors] Slow selection on 8.2Windows version  (Issue #1738)
                
                
  Hi @Isaina,
Please write the details:

What font do you use in your document?
How exactly is the text selection performed?
An example file so that the problem can be reproduced
A video showing the problem

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>




LuckySpecialist left a comment (ONLYOFFICE/DesktopEditors#1738)

Hi @Isaina,
Please write the details:

What font do you use in your document?
How exactly is the text selection performed?
An example file so that the problem can be reproduced
A video showing the problem

**Comment by LuckySpecialist** [comment]
Hi @Isaina!
Can you check the problem on the new version of the [editors](https://github.com/ONLYOFFICE/DesktopEditors/releases), is it still relevant to you?

**Comment by Isaina** [comment]
Yes with version 9.1 there is no speed rendering problem and the page or docx file is rendered correctly.
